src/components/_classes/component/editForm/utils.js

```diff
@@ -12,13 +12,13 @@ const EditFormUtils = {
       if (objValue.key === srcValue.key) {
         // Create complete objects by including missing keys.
         _.each(objValue, (value, prop) => {
-          if (!srcValue.hasOwnProperty(prop)) {
+          if (objValue.overrideEditForm || !srcValue.hasOwnProperty(prop)) {
             srcValue[prop] = value;
           }
         });
         _.each(srcValue, (value, prop) => {
-          if (!objValue.hasOwnProperty(prop)) {
-            objValue[prop] = value;
+          if (srcValue.overrideEditForm || !objValue.hasOwnProperty(prop)) {
+            srcValue[prop] = value;
```

Comment:
I believe this should still say... ```objValue[prop] = value;``` You are changing it to be ```srcValue``` and I don't think that is correct.

Reply:
fixed up! sorry, my fault.